### PR TITLE
Remove <beamed> from spec and by-example

### PR DESCRIPTION
**Pull request by adrianholovaty**
This pull requests removes `<beamed>` as a structural element — both from the spec and from "MNX By Example." See #193.

For now, it hasn't been replaced by any new/different approach. Eventually we'll need to determine a different way of specifying beaming, in a more presentational (less structural) way.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/mnx/pull/194.html" title="Last updated on Sep 28, 2020, 6:43 PM UTC (3ff5af0)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/mnx/194/f30a4e2...3ff5af0.html" title="Last updated on Sep 28, 2020, 6:43 PM UTC (3ff5af0)">Diff</a>

### CODE REVIEW

**Comment by adrianholovaty** [comment]
@dspreadbury @mdgood Could you review when you get a chance?

File: specification/common/index.bs [hunk 1]
```diff
@@ -3161,6 +3003,8 @@ notes in a swung triplet rhythm instead:
 ```
 </div>
 
+TODO: `<beamed>` no longer exists, so the previous example is invalid.
```
Comment (mdgood) [inline specification/common/index.bs:3006]: Could we just remove the previous example and then reword the introductory text to the next example? I think that should still cover the use of the performance-event while cleanly removing the beamed element from the spec.

**Comment by mdgood** [comment]
Looks good - just one suggestion regarding the TODO.

File: specification/common/index.bs (hunk 1 above)
Comment (adrianholovaty) [reply specification/common/index.bs]: @mdgood Thanks — I've just updated the pull request to handle that. Look OK?

**Review by mdgood — APPROVED** [review]

**Pull request merged**